.github/workflows/codeql.yml

```diff
@@ -0,0 +1,39 @@
+name: "CodeQL"
+
+on:
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
do we want to run multiple language checks in parallel?

Reply:
This comes from the default workflow template. For now, I think we won't check any other language, and, to be honest, I don't have a strong opinion about keeping this here in line 28.

Reply:
np: do you mind removing this matrix declaration?